src/app/src/main.cpp

```diff
@@ -136,8 +136,7 @@ int main(int argc, char* argv[]) {
         exit(0);
     }
 
-
-    std::string path = toml::find_or<std::string>(config, "nautilus", "outdir", userProfile.string());
+    std::string path = "E:\\";
```

Comment:
I kept getting segfaults when using 
`std::string path = toml::find_or<std::string>(config, "nautilus", "outdir", "E:\\");`

Reply:
you need to wrap the default value in a `std::string` constructor like the code below this

```cpp
auto prefix = toml::find_or<std::string>(config, "nautilus", "prefix", std::string("default_"));
```

Reply:
I would also add "E:\" to the config so it's obvious that users can change it if they want

Reply:
By the config, the nautilus.toml file?

Reply:
yeah, this one in the resources dir, https://github.com/CuriBio/nautilus_acquisition/blob/main/resources/nautilus.toml

That file gets copied to the installer and installed to Program Files, then copied to the user home directory when run for the first time

Reply:
Does it matter if in that file its `E:\` or `E:\\`, I tried getting the workspace to use the `nautilus/outdir` value set to `E:\test` in the `nautilus.toml` at `~/AppData/Local/Nautilus/nautilus.toml` but it just kept defaulting to `E:\`.

Reply:
I don't think it matters but you'll have to test it and see, to get `E:\\` you might have to make it `E:\\\\` because I think a single `\` needs escaping

Reply:
@jperson I was actually getting a build error from a merge conflict that I wasn't noticing, so it wasn't using my last changes. So this does all work and grabs the correct value in the `nautilus.toml` file.